test/test-cases/scale/saic/run-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+PYTHONPATH=. pytest -sv $@
```

Comment:
I would like to have a single entry point for all tests one day. Anyway all of them are going to use the same setup and the same connection protocols. Only the test content is different.

Reply:
@aputriax Agreed, I added an item to the DASH Test WG project tracker, which I invited you to. It's not quite the same as issues but good for tracking backlog items.